students/thorn/lesson04/dict_lab.py

```diff
@@ -0,0 +1,37 @@
+################################################################################
+# Dictionaries 1
+################################################################################
+def dictionaries_1():
+    # 1: Create dictionary.
+    chris_dict = {
+        'name':     'Chris',
+        'city':     'Seattle',
+        'cake':     'Chocolate'
+    }
+    # 2: Display dictionary.
+    print(chris_dict)
+    # 3: Delete cake entry from dictionary.
+    del chris_dict['cake']
+    # 4: Display dictionary.
+    print(chris_dict)
+    # 5: Add an entry for “fruit” with “Mango” and display the dictionary.
+    chris_dict['fruit'] = "Mango"
+    print(chris_dict)
+    # 6: Display the dictionary keys.
+    print(', '.join(chris_dict.keys()))
```

Comment:
note that it isn't necessary to call `.keys()` as it will iterate over dicts by default